src/App.jsx

```diff
@@ -1,10 +1,23 @@
 import { useState } from 'react'
 import './App.css'
+import Header from './components/Header'
+import Homepage from './components/Homepage'
+import { Route, Routes } from 'react-router-dom'
+import ArticleProvider from './components/ArticleProvider'
 
 function App() {
 
+  const [article, setCurrentArticle] = useState({});
```

Comment:
Looks like you aren't using this state???

Reply:
I'd initially planned to keep this in state within app due to rendering trending articles on the homepage but since I am no longer doing that, I have moved the state down into the single article component - thanks!